src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -315,12 +332,136 @@ private static void printLocales() {
             // print columns of 8
             if ((i + 1) % 8 == 0) {
                 ostream.println();
-                ostream.print(INDENT + INDENT);
+                ostream.print(TWOINDENT);
+            }
+        }
+        ostream.println();
+    }
+
+    private static void printSecuritySettings(String arg) {
+        switch (arg.toLowerCase(Locale.ROOT)) {
```

Comment:
Drop the conversion to lower case: none of the other categories allow "sloppy-case" inputs.